### PR TITLE
Add some guidelines for how to test well you code before creating PR

### PR DESCRIPTION
**Pull request by ravinderk**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -71,3 +71,12 @@ Follow the steps below to configure ESLint with PHPStorm:
 Open any JS file and you'll see suggestions from ESLint, if any.
 
 That's all! You're done.
+
+## How to test code before creating PR
+You can create a awesome PR if:
```
Comment (DevinWalker) [inline CONTRIBUTING.md]: "an awesome"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -71,3 +71,12 @@ Follow the steps below to configure ESLint with PHPStorm:
...
+1. `debug.log` is empty after testing code functionality.
```
Comment (DevinWalker) [inline CONTRIBUTING.md]: The `debug.log` found in `/wp-content/`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -71,3 +71,12 @@ Follow the steps below to configure ESLint with PHPStorm:
...
+2. Do not contain any testing purpose code.
```
Comment (DevinWalker) [inline CONTRIBUTING.md]: "Does not contain and testing code like `var_dump()` and `console.log()`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -71,3 +71,12 @@ Follow the steps below to configure ESLint with PHPStorm:
...
+3. Passes all phpunit test.
```
Comment (DevinWalker) [inline CONTRIBUTING.md]: "test"

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -71,3 +71,12 @@ Follow the steps below to configure ESLint with PHPStorm:
...
+4. No formatting issue while check formatting `php_codesniffer`/`eslint` with `WordPress Coding Standard`.
```
Comment (DevinWalker) [inline CONTRIBUTING.md]: "issues when checking formatting via"

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -71,3 +71,12 @@ Follow the steps below to configure ESLint with PHPStorm:
...
+5. Works fine at least on PHP `5.3` and PHP `7.*`
```
Comment (DevinWalker) [inline CONTRIBUTING.md]: "on at least"

**Comment by ravinderk** [comment]
@DevinWalker grammatical changes done.

**Pull request merged**